qa/suites/crimson-rados/thrash/workloads/small-objects.yaml

```diff
@@ -0,0 +1,21 @@
+overrides:
+  ceph:
+    crush_tunables: jewel
+tasks:
+- rados:
+    clients: [client.0]
+    ops: 400000
+    max_seconds: 600
+    max_in_flight: 64
+    objects: 1024
+    size: 16384
+    op_weights:
```

Comment:
We probably want to disable everything but reads and writes.

Reply:
@athanatos  disabled and repushed.